inc/listings.php

```diff
@@ -33,6 +33,19 @@ function cptui_listings_assets() {
 	}
 
 	wp_enqueue_style( 'cptui-css' );
+	wp_enqueue_script( 'cptui-listings-delete' );
+
+	$core                  = get_post_types( [ '_builtin' => true ] );
+	$public                = get_post_types( [ '_builtin' => false, 'public' => true ] );
+	$private               = get_post_types( [ '_builtin' => false, 'public' => false ] );
+	$registered_post_types = array_merge( $core, $public, $private );
+
+	wp_localize_script( 'cptui-listings-delete', 'cptui_type_data',
+		[
+			'confirm'             => esc_html__( 'Are you sure you want to delete this? Deleting will NOT remove created content.', 'custom-post-type-ui' ),
+			'existing_post_types' => $registered_post_types,
+		]
+	);
```

Comment:
Feel free to correct me if parts of this is just early work and perhaps initial copy/paste, but are we needing the post type information? From what I've tested so far, we only really need the confirmation text.